Userland/Services/WebContent/ConnectionFromClient.h

```diff
@@ -46,8 +46,8 @@ class ConnectionFromClient final
 private:
     explicit ConnectionFromClient(NonnullOwnPtr<Core::LocalSocket>);
 
-    Web::Page& page();
-    Web::Page const& page() const;
+    PageClient& page(u64 index = 0);
+    PageClient const& page(u64 index = 0) const;
```

Comment:
How much more work is it to get rid of the `= 0` here?

Reply:
here, not much, if "get rid of"  means just moving it into the cpp file. Getting rid of the assumption that we want page 0? I don't know yet. I think it involves passing that information back to the UI chrome and modifying the IPC definitions.

Reply:
Yeah, I meant getting rid of the assumption and being explicit about the ID everywhere. :)